### PR TITLE
cell/row optimizations, group member bug fix, cross sheet reference addition

### PR DESCRIPTION
**Pull request by mishaberman**

### CODE REVIEW

File: Smartsheet.Net.Standard/Http/SmartsheetHttpClient.cs [hunk 1]
```diff
@@ -549,14 +553,23 @@ public async Task<Sheet> CopySheet(string newName, long? sourceSheetId, long? de
...
+                throw new Exception("Sheet ID cannot be null");
+            }
+
+            string optionsString = String.Empty;
+
+            if(options != null && options.Any()){
+                optionsString = $"?{String.Join("&", options)}";
+            }
+
             if (sheetId == null)
```
Comment (rgdick11) [inline Smartsheet.Net.Standard/Http/SmartsheetHttpClient.cs]: This check is in here twice now. Remove this second one.

File: Smartsheet.Net.Standard/Http/SmartsheetHttpClient.cs [hunk 2]
```diff
@@ -992,7 +1022,7 @@ public async Task<ISmartsheetObject> AddUser(User user, string accessToken = nul
         {
             this._HttpClient.DefaultRequestHeaders.Add("sendEmail", "false");
 
-            var response = await this.ExecuteRequest<ResultResponse<User>, User>(HttpVerb.POST, string.Format("users"), data: user, accessToken: accessToken);
+            var response = await this.ExecuteRequest<ResultResponse<User>, User>(HttpVerb.POST, string.Format("users?allowInviteUnlicensedTrialUser=true"), data: user, accessToken: accessToken);
```
Comment (rgdick11) [inline Smartsheet.Net.Standard/Http/SmartsheetHttpClient.cs]: Remove this change - not meant to be here

File: Smartsheet.Net.Standard/Entities/Cell.cs [hunk 3]
```diff
@@ -69,6 +69,8 @@ public Cell Build(bool? strict)
         public string Formula { get; set; }
 
 		public bool? Strict { get; set; }
+		
+		public bool? OverrideValidation { get; set; }
```
Comment (rgdick11) [inline Smartsheet.Net.Standard/Entities/Cell.cs]: Where is this used?

Comment (rgdick11) [reply Smartsheet.Net.Standard/Entities/Cell.cs]: Nevermind, it's apparently part of the Cell object in the API. Carry on.

File: Smartsheet.Net.Standard/Interfaces/ISmartsheetHttpClient.cs [hunk 4]
```diff
@@ -59,9 +59,15 @@ public interface ISmartsheetHttpClient
 
 		//	Columns
 		Task<Column> EditColumn(long? sheetId, long? columnId, Column model, string accessToken = null);
+		Task<Column> CreateColumn(long? sheetId, Column model, string accessToken = null);
```
Comment (rgdick11) [inline Smartsheet.Net.Standard/Interfaces/ISmartsheetHttpClient.cs]: Add DeleteColumn() to the interface

**Comment by mishaberman** [comment]
Added more interface methods for http client. Code should be ready to review and merge. Thanks!

**Pull request merged**